### DISCUSSION TITLE
Is it possible to request for a name from an inactive account?

### DISCUSSION THREAD

**Discussion by NotSinan** [community/community#36944] [Other Feature Feedback, Questions, & Ideas] [answered]
Can I request to get a name from an account that is inactive?

**Comment by airtower-luna** [comment, marked as answer]
Simple answer: No.

You can read a detailed statement from a staff member here: https://github.com/orgs/community/discussions/28516#discussioncomment-3337421

**Reply by NotSinan** [reply]
Thank you for the clarification.

**Comment by ossanna16** [comment]
Hi there @p-oOasddad and welcome to our community 😄. Thanks so much for asking a great question!
